modules/samples/s0/gadget/usa-population-by-age-and-gender/index.xml

```diff
@@ -11,12 +11,12 @@
         <![CDATA[
 
             <!-- Load the stylesheet-->
-            <link rel="stylesheet" href="/portal/css/bootstrap.min.css">
+            <link rel="stylesheet" href="/portal/libs/bootstrap_3.3.5/css/bootstrap.min.css">
             <link rel="stylesheet" type="text/css" href="content/index.css">
 
             <!-- Loading common javascript libraries -->
             <script type="text/javascript" src="js/jquery-1.11.3.min.js"></script>
-            <script language="javascript" type="text/javascript" src="/portal/js/bootstrap.min.js"></script>
+            <script language="javascript" type="text/javascript" src="/portal/libs/bootstrap_3.3.5/js/bootstrap.min.js"></script>
```

Comment:
The language attribute has been deprecated for a long time, and should not be used. https://www.w3.org/TR/html5/scripting-1.html#attr-script-type